website/docs/blueprints/ai-ml/bionemo.md

```diff
@@ -0,0 +1,225 @@
+:::
+
+## Introduction
+
+[NVIDIA Bionemo](https://www.nvidia.com/en-us/clara/bionemo/) is a generative AI platform for drug discovery that simplifies and accelerates the training of models using your own data and scaling the deployment of models for drug discovery applications. BioNeMo offers the quickest path to both AI model development and deployment, accelerating the journey to AI-powered drug discovery. It has a growing community of users and contributors, and is actively maintained and developed by the NVIDIA.
+
+Since Bionemo is containerized, we can deploy it to Amazon Sagemaker, AWS ParallelCluster, Amazon ECS and Amazon EKS. In this solution we are going to focus on running Bionemo on Amazon EKS.
+
+*Source: https://blogs.nvidia.com/blog/bionemo-on-aws-generative-ai-drug-discovery/*
+
+## Bionemo on Kubernetes
+
+In order to deploy Bionemo on Kubernetes, we need 3 major components.
+
+1) [**Kubeflow Training Operator**](https://www.kubeflow.org/docs/components/training/)
+2) [**NVIDIA Device Plugin**](https://github.com/NVIDIA/k8s-device-plugin)
+3) [**FSx for Lustre CSI Driver**](https://docs.aws.amazon.com/eks/latest/userguide/fsx-csi.html)
+
+
+## Deploying the Example
+
+In this example we are going to deploy an Amazon EKS cluster and run a data preparation job and a distributed model training job.
+
+<CollapsibleContent header={<h3><span>Pre-requisites</span></h3>}>
+
+Ensure that you have installed the following tools on your machine.
+
+1. [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+2. [kubectl](https://Kubernetes.io/docs/tasks/tools/)
+3. [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+4. [python3](https://www.python.org/)
+
+</CollapsibleContent>
+
+<CollapsibleContent header={<h3><span>Deploy the EKS Cluster with KubeRay Operator</span></h3>}>
+
+#### Clone the repository
+
+```bash
+git clone https://github.com/awslabs/data-on-eks.git
+```
+
+#### Initialize Terraform
+
+Navigate into the example directory
+
+```bash
+cd data-on-eks/ai-ml/bionemo
+```
+
+#### Run the install script
+
+Use the provided helper script `install.sh` to run the terraform init and apply commands. By default the script deploys EKS cluster to `us-west-2` region. Update `variables.tf` to change the region. This is also the time to update any other input variables or make any other changes to the terraform template.
+
+
+```bash
+./install .sh
+```
+
+</CollapsibleContent>
+
+<CollapsibleContent header={<h3><span>Verify Deployment</span></h3>}>
+
+Update local kubeconfig so we can access kubernetes cluster
+
+```bash
+aws eks update-kubeconfig --name bionemo-cluster #or whatever you used for EKS cluster name
```

Comment:
name of the cluster is not matching with the varaibale name. change this to `bionemo-on-eks`